float32 pitch
float32 roll
float32 yaw
float32 velocity
float32 duration
string units